web/themes/custom/pineapple/sass/layouts/nodes/_recipe.scss

```diff
@@ -54,15 +53,22 @@
       width: 28%;
       margin-left: 22px;
       margin-top: 0;
+      max-height: 350px;
     }
 
     .author-image-container {
       display: flex;
-      justify-content: center;
       padding: $unit *2;
+      width: 20%;
```

Comment:
View profile image on mobile (320px) itss waaaaaay to small and needs to be a lot bigger, half the size of it's container at a *minimum*

Reply:
Okay. Done.